antsibull/write_docs.py

```diff
@@ -175,13 +179,16 @@ async def write_plugin_lists(collection_name: str,
 
 
 async def output_indexes(collection_info: CollectionInfoT,
-                         dest_dir: str) -> None:
+                         dest_dir: str,
+                         toplevel_collection_index: t.Optional[bool] = True) -> None:
```

Comment:
I'd rather the caller just made the decision to call two functions or one function instead of pushing a boolean into the parameters.  I think that will be okay as including the toplevel index in this function may have been a premature optimization (it allows writing it in parallel with all the other indexes but it's only a single file to wait on).

Reply:
I split up the function in bc27cef654990533348be988e2d56d084e9502a9.